### PR TITLE
add jbang labelling

### PR DESCRIPTION
**Pull request by maxandersen**

### CODE REVIEW

File: .github/boring-cyborg.yml [hunk 1]
```diff
@@ -144,9 +144,12 @@ labelPRBasedOnFilePath:
...
     area/jaxb:
         - extensions/jaxb/**/*
+    area/jbang:
+        - **/*JBang*
+        - **/*jbang*
     area/kafka:
-        - extensions/kafka-client/**/*
-        - integration-tests/kafka/**/*
+    - extensions/kafka-client/**/*
+    - integration-tests/kafka/**/*
```
Comment (gsmet) [inline .github/boring-cyborg.yml:152]: This change is weird? Doesn't look consistent with the surrounding items?

Comment (maxandersen) [reply .github/boring-cyborg.yml:152]: hmm - gah - stuipd vscode changes indents on paste.fixing.

Comment (maxandersen) [reply .github/boring-cyborg.yml]: force pushed.

**Review by gastaldi — APPROVED** [review]

File: .github/autoissuelabeler.yml [hunk 2]
```diff
@@ -20,6 +20,9 @@
 - labels: [area/maven]
   title: "(?i).*maven.*"
   notify: [quarkusio/devtools]
+- labels: [area/jbang]
+  title: "(?i).*jbang.*"
+  notify: [quarkusio/devtools,maxandersen]
```
Comment (gastaldi) [inline .github/autoissuelabeler.yml:25]: Aren't you already in the `quarkusio/devtools` group?

Comment (maxandersen) [reply .github/autoissuelabeler.yml:25]: true - its probably redundant ;)

Comment (maxandersen) [reply .github/autoissuelabeler.yml:25]: but wanted to add in to see if it changes the notification setup i have. i'll remove it if the false positive gets too much.

**Pull request merged**